### DISCUSSION TITLE
API key not working 1first configuration

### DISCUSSION THREAD

**Discussion by doriss40** [jeroenterheerdt/HAsmartirrigation#410] [Q&A]
Sorry to bother you all, 
I have to work on something simple but I can’t pass the first configuration screen...
The key was created 10 days ago and I tried another one...
Can someone give me a little help?

![2024-06-14_13-16-25](https://github.com/jeroenterheerdt/HAsmartirrigation/assets/170401537/df2e2ab7-9f3b-4598-bd0c-44da0bb34968)

**Comment by vagy66** [comment]
I have the same issue

**Comment by jeroenterheerdt** [comment]
Did you read and follow this? https://github.com/jeroenterheerdt/HAsmartirrigation?tab=readme-ov-file#getting-open-weather-map-api-key

Note that you need to enter credit card info as well.
Also, which version are you on?

**Comment by doriss40** [comment]
Hello sorry for that but i'm unable to find where i have to put the credit card on open weather map. 
When i registered for the free api key i never had to fill it and i'm unable to find in the menus where i can change that.


Smart Irrigation v2024.4.3

Core 2024.6.3
Supervisor 2024.06.0
Operating System 12.3
Interface utilisateur 20240610.1

**Comment by jeroenterheerdt** [comment]
You enter the credit card info at owm:

OpenWeatherMap API V2.5 will be closed in June 2024. After this date, you will need to use API V3.0.

To continue using the service:

Visit the OpenWeatherMap website and activate the One Call subscription.

During activation, you will be prompted for a credit card, but you will not be charged unless you exceed the free tier limits.

**Comment by doriss40** [comment]
Ok it works sorry for that, i had to re-suscribe !
